rosbag2_transport/include/rosbag2_transport/play_options.hpp

```diff
@@ -48,6 +49,11 @@ struct PlayOptions
 
   // Sleep before play. Negative durations invalid. Will delay at the beginning of each loop.
   rclcpp::Duration delay = rclcpp::Duration(0, 0);
+
+  bool disable_keyboard_controls = false;
+  // keybindings
+  KeyboardHandlerBase::KeyCode pause_resume_toggle_key = KeyboardHandlerBase::KeyCode::SPACE;
+  KeyboardHandlerBase::KeyCode play_next_key = KeyboardHandlerBase::KeyCode::S;
```

Comment:
IMO it would be more naive to use `KeyboardHandlerBase::KeyCode::CURSOR_RIGHT`  for play next command.

Reply:
That seems good to me too. I was thinking about arrow keys for faster/slower rate, but that could use UP/DOWN